Documented the usage of react-onclickoutside on an SFC (#293)
Pomax/react-onclickoutside: README.md

);
```

> Besides having `handleClickOutside(evt)` class method, there are two other ways to pass a custom event handler:

Reviewer: ```suggestion
In addition to specifying a `handleClickOutside(evt)` class method, there are two alternative ways to pass in a custom event handler:
```

> * either specify the function to be used by the HOC in the configuration object (second parameter)

Reviewer: ```suggestion
* you can specify which function should be used by the HOC, by passing indicating which function should be used in the configuration object, or
```

> * or pass function to be used by the HOC as a prop during the usage of the wrapped component

Reviewer: ```suggestion
* you can pass the function to be used by the HOC as a component property `handleClickOutside={...}` in your JSX
```

};
var EnhancedComponent = onClickOutside(MyComponent, clickOutsideConfig);
```
or:

> ```js

Reviewer: I would recommend changing the structure here, too. Rather than stating that there are two ways and then giving a list, I would advocate:
```
There are two ways. The first is to do X:

<<code here>>

The second is Y:

<<code here>>
```

// load the HOC:
import React, { Component } from "react";
import onClickOutside from "react-onclickoutside";

// create a new component, wrapped below by onClickOutside HOC:
function MyComponent(props) {
  // ...
}

var EnhancedComponent = onClickOutside(MyComponent);

// ...handling code should be passed during EnhancedComponent usage:

> <EnancedComponent handleClickOutside={() => {

Reviewer 2: _EnhancedComponent_

Author: I've fixed it. Thanks.
